cmd/clairctl/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
How many outside contributors do you have? Is it possible to relicense this to Apache2 to be uniform with the rest of the repository?